cmd/update/cluster/flag.go

```diff
@@ -0,0 +1,61 @@
+package cluster
+
+import (
+	"github.com/giantswarm/microerror"
+	"github.com/spf13/cobra"
+	"k8s.io/cli-runtime/pkg/genericclioptions"
+)
+
+const (
+	flagName           = "name"
+	flagReleaseVersion = "release-version"
+	flagScheduledTime  = "scheduled-time"
+	flagProvider       = "provider"
+)
+
+type flag struct {
+	config         genericclioptions.RESTClientGetter
+	print          *genericclioptions.PrintFlags
+	Name           string
+	ReleaseVersion string
+	ScheduledTime  string
+	Provider       string
+}
+
+func (f *flag) Init(cmd *cobra.Command) {
```

Comment:
I think the `--namespace` flag is missing here.

Reply:
This is some magic because we use `kubectl` which I recently learned it's injected magically, so it we don't have to specify it.

Reply:
Oh right. :) It slipped my mind.